### ISSUE TITLE
Testing #5764: a11y --> no focus on highlight button

### ISSUE THREAD

**Issue by sabinammm** [liqd/a4-kosmo#228]
**URL:** https://kosmo-dev.liqd.net/userdashboard/moderation/detail/deliberation-on-how-to-use-ai-in-discussions/
**user:** moderator
**expected behaviour:** While tabbing through the Moderaton Dashboard, I know where I'm at b/c the place/button is marked by a border
**behaviour:** Some buttons in the dashboard on the notification tile have a focus border, but most do not. There's no focus on the highlight (neither on archive-functionality, 'add statement', and 'block' buttons). There is a focus on participant name and comment.
**important screensize:** any
**device & browser:** any
**Comment/Question:** 

Screenshot?
![image](https://user-images.githubusercontent.com/17978375/167412168-905e0523-a94e-4d16-b3eb-baacabc25868.png)
